Automattic/jetpack
# Admin menu: update with latest version of Jetpack logo

Fixes #29367

## Proposed changes:

This involves a few changes:

1. Remove the Jetpack icon font we were using until today.
2. Remove the stylesheets that were enqueued to add that icon font.
    * That stylesheet also included some CSS for the Contact Form icon (see #17290). Now that we've switched to a dashicon for the Contact Form, it may no longer be necessary.
3. Add a new method to the Logo package, returning a base64-encoded SVG for the Jetpack icon.
4. Use that new method everywhere we need a Jetpack Logo in an admin menu.

<img width="330" alt="image" src="https://user-images.githubusercontent.com/426388/224366531-8616c42d-0f34-4f3e-8bd7-8f76f7732353.png">

<img width="382" alt="image" src="https://user-images.githubusercontent.com/426388/224367624-168e1fba-7ed5-40d5-b791-339d0c94862f.png">

<img width="372" alt="image" src="https://user-images.githubusercontent.com/426388/224391114-3c153759-f311-40d6-915e-e7c792306117.png">


### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

This will require testing on different sites:

1. On a WoA site so you can test nav-unification (the admin menu displayed in Calypso).
2. On a self-hosted site using the Jetpack plugin.
3. On a self-hosted site using a standalone plugin.

In all 3 sites, the Jetpack icon should look good regardless of your admin color scheme (set under Users > My profile), and whether you're in the Jetpack menu (e.g. looking at Jetpack > Dashboard) or on another admin page. On the WoA site and the site running the Jetpack plugin, you should also check the Feedback menu icon.
